src/components/dictionaryConcepts/components/ViewMappingsModal.jsx

```diff
@@ -67,19 +64,14 @@ export class ViewMappingsModal extends Component {
                         editMapping,
                         concepts,
                         mappings,
-                        buttonName: 'Edit',
+                        buttonName: 'View',
```

Comment:
Is this change part of the ticket?

Reply:
Yes, it was part of this ticket, I replaced the edit button with view as the edit function was removed.

After getting the response from Darius concerning this ticket, I will have to increase the scope of what the ticket handles. I need to remove the View mapping button that leads to the modal, I will need to remove the entire modal too.

Can I do that within this ticket also?

Reply:
Remove as much as required by the ticket.

Reply:
I have updated my changes